xGame/Java/xgame/pom.xml

```diff
@@ -44,6 +44,11 @@
 			</exclusion>
 		</exclusions>
 	</dependency>
+	<dependency>
+    	<groupId>com.google.guava</groupId>
+    	<artifactId>guava</artifactId>
+    	<version>20.0</version>
+	</dependency>
```

Comment:
What is this for?

Reply:
This is the package I'm using for password hashing